src/script/util/Environment.ts

```diff
@@ -38,6 +47,44 @@ const isLocalhost = (): boolean => [APP_ENV.LOCALHOST, APP_ENV.VIRTUAL_HOST].inc
 const isProduction = (): boolean => {
   return window.wire.env.ENVIRONMENT === BackendEnvironment.PRODUCTION;
 };
+export const getWebEnvironment = () => {
+  const appBase = window.wire.env.APP_BASE;
+  const environment = {
+    isEdge: false,
+    isStaging: false,
+    isDev: false,
+    isInternal: false,
+    isLinked: false,
+    isLocalhost: false,
+    isProduction: false,
+    name: '',
+  };
+
+  if (appBase.includes(DEV_ENVIRONMENT_IDENTIFIERS.LOCAL || isLocalhost())) {
```

Comment:
And maybe we can use for this case swtich/case method ? 🤔 
I think it should be like that :D 

```suggestion
  if (appBase.includes(DEV_ENVIRONMENT_IDENTIFIERS.LOCAL) || isLocalhost()) {
```

Reply:
I like if else more, since we iterate not just over one object